index.html

```diff
@@ -279,5 +279,34 @@ <h3 class="titleA">About My self</h3>
                 </li>
               </ul>
             </section>
-  
-  
\ No newline at end of file
+            <!--Works Section contact page-->
+            <section class="page5">
+              <div class="@continer">
+                <div class="bag">
+               
+                <div class="h1">
+                  <form>
+                  <ul><li class="h2"> <h2>Contact me</h2></li>
+                  <li class="p1"> <p>If you have an application you<br>are interested in developing, a <br>feature that you need built or a <br>project that needs coding.<br> I’d love to help with it!</p></li></ul>
+                  <div class="warpare">
+                    <input type="User name" required="required">
+                    <label for="">User name:</label>
+                  </div>
+                      <div class="salar">
+                        <input type="email" required="required">
+                          <label for="">Email:</labe></div>
+                            <div class="sayed">
+                              <input type="User name" required="required">
+                              <label for="">Write your message here</label>
```

Comment:
>Make sure that:
    - Name field accepts up to 30 chars.
    - Message field accepts up to 500 chars.

- [OPTIONAL] This was not pointed out by the previous reviewers but looking at the project requirement, the name input is meant to have a maximum character attribute, `maxlength="30"`, I'll suggest you update this. :+1: 
- [OPTIONAL] Also the message input is meant to be a text area `<textarea>` and also have a maximum character length attribute,  `maxlength="500"` I'll suggest you update this too. :+1: :+1: :smile: